cli/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@xata.io/xata",
+  "version": "0.0.0",
+  "description": "Xata.io CLI",
+  "author": "Xata Inc.",
+  "bin": {
+    "xata": "./bin/run"
+  },
+  "homepage": "https://github.com/xataio/client-ts",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "type": "module",
+  "repository": "xataio/client-ts",
+  "files": [
+    "/bin",
+    "/dist",
+    "/npm-shrinkwrap.json",
+    "/oclif.manifest.json"
+  ],
+  "dependencies": {
+    "@oclif/core": "^1.8.1",
+    "@oclif/plugin-help": "^5",
+    "@oclif/plugin-plugins": "^2.0.1"
+  },
+  "devDependencies": {
+    "chai": "^4.3.6",
+    "eslint": "^7.32.0",
+    "eslint-config-oclif": "^4",
+    "eslint-config-oclif-typescript": "^1.0.2",
+    "globby": "^11",
+    "oclif": "^3",
+    "shx": "^0.3.3",
+    "ts-node": "^10.2.1",
+    "tslib": "^2.3.1",
+    "typescript": "^4.4.3"
+  },
+  "oclif": {
+    "bin": "xata",
+    "dirname": "xata",
+    "commands": "./dist/commands",
+    "plugins": [
+      "@oclif/plugin-help",
+      "@oclif/plugin-plugins"
+    ],
+    "topicSeparator": " ",
+    "topics": {
+      "hello": {
+        "description": "Say hello to the world and others"
+      }
+    }
+  },
+  "scripts": {
```

Comment:
The files in ``bin`` probably make sense as scripts. I couldn't get to work ``dev`` one, but ``run`` works fine

Reply:
Cool. I've done that but I've removed the `dev` one because it doesn't read correclty the ESM commands and I think we don't need it.